gcp/modules/couchdb/resources/couchdb-network-policy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: couchdb-network-policy
```

Comment:
It seems that the ports 5984 and 5986 are also used in a cluster setup.
http://docs.couchdb.org/en/stable/cluster/setup.html#firewall

Reply:
Unfortunately, I didn't see port `5986` exposed either in the pod spec for couchdb or even on the running containers. This might be another one of those deployment differences between `aws` cluster and `gcp` clusters. When I run through my next deployment-from-scratch, I'll be on the lookout for this specifically and report what I find.

Reply:
We should know exactly what are the ports used for and if they need to be accessed and what from (clients vs cluster nodes).

```
Chart exposes these:
            - name: couchdb
              containerPort: 5984
            - name: epmd
              containerPort: 4369
            - containerPort: 9100
```

Reply:
Yes, we use the 5986 port to setup the cluster in AWS. I don't know why it is not used in GCP. I guess that we will realize of this kind of requirements once we improve the 'couchdb-assembler' in GPII-3343.

Reply:
this is a copy from CouchDB docs (http://docs.couchdb.org/en/stable/config/http.html):

> In CouchDB 2.x, the httpd section mostly refers to the node-local port, on port 5986 by default. This port is used only for maintenance and administrative tasks. It should not be used for regular CouchDB access, and for security reasons, should always be bound to localhost (127.0.0.1) or a private LAN segment only.